docs/components/buttons/index.hbs

```diff
@@ -0,0 +1,90 @@
+---
+title: Buttons
+description: Showcases and descriptions of the different types of buttons and how to use them. 
+nav: side/components/buttons
+order: -1
+toc:
+  - title: Standard Button
+    slug: standard-buttons
+  - title: Icon Button
+    slug: icon-buttons
+  - title: AXA Button
+    slug: axa-buttons
+  - title: Ghost Button
+    slug: ghost-buttons
+---
+
+{{#> page}}
+  {{#*inline "content"}}
+    <h1 class="h2">{{title}}</h1>
+    <p>
+      Our buttons are based completely on the default bootstrap buttons.
+    </p>
```

Comment:
Feel free to add the `.lead` class to the first paragraph, that usually quickly summarizes the following page.